### DISCUSSION TITLE
Review the dev/development branch

### DISCUSSION THREAD

**Discussion by danielRep** [bao-project/bao-docs#18] [General]
To @josecm & @AfonsoSantos96 

I kindly ask each of you to review the `dev/development` branch to see if there is still any changes that need to be committed in a single feature branch and PR'ed. I remember that this branch is one of the initial ones that we started using for "development" before we had a defined contribution process. 

Thank you.

**Comment by josecm** [comment]
It seems the 'coding style' and  'code documentation' sections are missing.  I'll submit a PR for coding style. @AfonsoSantos96 can you submit a separate PR for the code documentation guidelines?

**Reply by josecm** [reply]
Both the code documentation and coding style PRs were submitted. Respectively, #39 and #52. The first was accepted, and the second is still a draft.
